_includes/manuals/1.10/4.3.6_smartproxy_puppet.md

```diff
@@ -124,6 +124,58 @@ The puppetssh provider uses SSH to connect to the client using SSH keys and run
 #:puppetssh_keyfile: /etc/foreman-proxy/id_rsa
 </pre>
 
+Here is a quick howto to enable puppetssh on your server.
+
+On the `foreman-proxy server`, enable the puppetssh provider and uncomment user and ssh keys settings:
+<pre>
+sed -i 's!^#\?\(:puppet_provider:\).*$!\1 puppetssh!' \
+  /etc/foreman-proxy/settings.d/puppet.yml
+sed -i 's!^#\?\(:puppetssh_user:\).*$!\1 root!' \
+  /etc/foreman-proxy/settings.d/puppet.yml
+sed -i 's!^#\?\(:puppetssh_keyfile:\).*$!\1 /etc/foreman-proxy/id_rs!' \
+  /etc/foreman-proxy/settings.d/puppet.yml
+</pre>
+
+We also need to actually generate this new SSH key:
+<pre>
+ssh-keygen -t rsa -N '' -f /etc/foreman-proxy/id_rsa
+chgrp foreman-proxy /etc/foreman-proxy/id_rsa
+chmod 640 /etc/foreman-proxy/id_rsa
+</pre>
+
+Then, foreman-proxy should be restarted. If using systemd, run:
+<pre>systemctl restart foreman-proxy</pre>
+Else, the good old service command should work:
+<pre>service foreman-proxy restart</pre>
+
+After that, you should make sure the user running foreman-proxy service has a valid home and a writable `.ssh subdirectory`.
+Check defined home directory by running
+<pre>
+ls -d ~foreman-proxy
+</pre>
+
+It should return `/usr/share/foreman-proxy`. Now we will create a .ssh subdirectory and set the proper permissions:
+<pre>
+mkdir /usr/share/foreman-proxy/.ssh
+chown foreman-proxy:foreman-proxy /usr/share/foreman-proxy/.ssh
+chmod 700 /usr/share/foreman-proxy/.ssh
+</pre>
+
+An additional step, but not required (`security warning`) is to disable host keys checking. It means any host allowing this SSH keys to connect will be reachable, without the need to run ssh root@client as foreman-proxy user to accept the host key. The goal here is to allow automatic deployment by only population the `authorized_keys` file on new clients.
+
+To enable such behavior, we will create a SSH client configuration file disable host key checking and not storing that kind of keys:
```

Comment:
"[..] configuration file to diasble host key checking, and disable storing of public keys."